po/pt.po

```diff
@@ -0,0 +1,739 @@
+
+#: data/resources/ui/shortcuts.ui:20
+msgctxt "shortcut window"
+msgid "Show Shortcuts"
+msgstr "Mostrar atalhos"
+
+#: data/resources/ui/shortcuts.ui:26
+msgctxt "shortcut window"
+msgid "Quit"
+msgstr "Fechar"
+
+#: data/resources/ui/shortcuts.ui:34
+msgctxt "shortcut window"
+msgid "Color"
+msgstr "Cor"
+
+#: data/resources/ui/shortcuts.ui:37
+msgctxt "shortcut window"
+msgid "Pick a new Color"
+msgstr "Selecionar uma nova cor"
+
+#: data/resources/ui/shortcuts.ui:43
+msgctxt "shortcut window"
+msgid "Randomize the Color"
+msgstr "Gerar uma cor aleatória"
+
+#: data/resources/ui/window.ui:6
+msgid "_Clear History"
+msgstr "_Limpar histórico"
+
+#: data/resources/ui/window.ui:10
+msgid "_Random Color"
+msgstr "_Gerar cor aleatória"
+
+#: data/resources/ui/window.ui:16
+msgid "_Preferences"
+msgstr "_Preferências"
+
+#: data/resources/ui/window.ui:20
+msgid "_Keyboard Shortcuts"
+msgstr "_Atalhos do teclado"
+
+#: data/resources/ui/window.ui:24
+msgid "_About Eyedropper"
+msgstr "_Sobre o Eyedropper"
+
+#: data/resources/ui/window.ui:54
+msgid "No Color"
+msgstr "Nenhuma cor selecionada"
+
+#: data/resources/ui/window.ui:55
+msgid "Select a color from the screen to get started."
+msgstr "Selecione uma cor na tela para começar"
+
+#: data/resources/ui/window.ui:58
+msgid "_Pick a Color"
+msgstr "_Selecionar cor"
+
+#: data/resources/ui/window.ui:96
+msgctxt "Tooltip of the colorpicker button"
+msgid "Pick a Color"
+msgstr "Selecionar uma nova cor"
+
+#: data/resources/ui/window.ui:103
+msgid "Main Menu"
+msgstr "Menu principal"
+
+#: data/resources/ui/window.ui:129
+msgctxt "Tooltip of the palette button"
+msgid "Open Palette Window"
+msgstr "Abrir o menu da paleta"
+
+#: data/resources/ui/window.ui:152
+msgid "Copy Name"
+msgstr "Copiar Nome"
+
+#: data/resources/ui/window.ui:158
+msgid "Copy Hex Code"
+msgstr "Copiar código HEX"
+
+#: data/resources/ui/window.ui:163
+msgid "Copy RGB"
+msgstr "Copiar código RGB"
+
+#: data/resources/ui/window.ui:168
+msgid "Copy HSL"
+msgstr "Copiar código HSL"
+
+#: data/resources/ui/window.ui:173
+msgid "Copy HSV"
+msgstr "Copiar código HSV"
+
+#: data/resources/ui/window.ui:178
+msgid "Copy CMYK"
+msgstr "Copiar código CMYK"
+
+#: data/resources/ui/window.ui:183
+msgid "Copy XYZ"
+msgstr "Copiar código XYZ"
+
+#: data/resources/ui/window.ui:189
+msgid "Copy CIELAB"
+msgstr "Copiar código CIELAB"
+
+#: data/resources/ui/window.ui:195
+msgid "Copy HWB"
+msgstr "Copiar código HWB"
+
+#: data/resources/ui/window.ui:201
+msgid "Copy CIELCh / HCL"
+msgstr "Copiar código CIELCh / HCL"
+
+#: data/resources/ui/window.ui:207
+msgid "Copy LMS"
+msgstr "Copiar código LMS"
+
+#: data/resources/ui/window.ui:213
+msgid "Copy Hunter Lab"
+msgstr "Copiar código Hunter Lab"
+
+#. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
+#: src/widgets/about_window.rs:51
+msgid "translator-credits"
+msgstr "swyknox"
```

Comment:
While this is technically not wrong, are you sure that you do not want to link an email address or website (or your GitHub account)?

Reply:
Well thought out, I added my email!